includes/definitions/alliedtelesis.yaml

```diff
@@ -1,10 +1,14 @@
-os: allied
-text: AlliedWare
+os: alliedtelesis
+text: 'AW+'
 type: network
+icon: alliedtelesis
+group: alliedtelesis
 ifname: 1
 over:
     - { graph: device_bits, text: Traffic }
 discovery:
     -
       sysObjectId: .1.3.6.1.4.1.207.
       sysObjectId_except: .1.3.6.1.4.1.207.1.4.126
+    - sysDescr: 'Allied Telesis'
```

Comment:
Is this needed. You test file uses `.1.3.6.1.4.1.207.1.14.137` which would be picked up by the original check.